src/chm/documents/customactions/qtexec.html.md

```diff
@@ -23,14 +23,29 @@ When the QtExec action is run as an immediate custom action, it will try to exec
 
 This will result in running the command line in the immediate sequence. If the exit code of the command line in this example indicates an error (meaning that the return code is not equal to 0) then the setup will fail because the Return value is set to &ldquo;check.&quot; Changing the Return value to &quot;ignore&quot; will cause the setup to log the failure but skip it and continue instead of failing the entire setup.
 
-If you want to run more than one command line in the immediate sequence then you will need to schedule multiple QtExec custom actions and set the QtExecCmdLine property to a new value by scheduling a property-setting custom action immediately before each instance of the QtExec custom action.
+If you want to run more than one command line in the immediate sequence then you will need to schedule multiple QtExec custom actions and set the WixQuietExecCmdLine property to a new value by scheduling a property-setting custom action immediately before each instance of the QtExec custom action.
+
+## Silent execution
+
+If you need to run a program without logging any of the input parameters or output of the executable, for example for security or privacy reasons, you want WixSilentExec:
```

Comment:
comma splice. You can put the whole "for example" phrase in --- em dashes. And then put a comma after "for example".